### PR TITLE
Refactor buy button behavior and change storefront labels

### PR DESCRIPTION
**Pull request by theomoura**
#### What is the purpose of this pull request?
Change buy button behavior for mobile and web devices. The buy button should appear on mobile devices when 'Always Show' option is selected and disappear on the other options.

#### What problem is this solving?
Buy button was never showing on mobile devices and somo storefront options only works on web devices.

#### How should this be manually tested?
[Access this workspace](https://adminshelffix--storecomponents.myvtex.com/admin/cms/storefront)

#### Screenshots or example usage
clipboard 2019-07-01 at 11 25 45 am 1

#### Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/index.js [hunk 1]
```diff
@@ -316,11 +311,11 @@ class ProductSummary extends Component {
     })
 
     const informationClasses = classNames(`${productSummary.informations}`, {
```
Comment (JuanAlmeida12) [inline react/index.js]: `${productSummary.informations}` can be only `productSummary.informations`

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,11 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Fixed 
+- Buy button always showing with on hover option.
+### Changed
+- Display buy button on mobile devices.
```
Comment (vitoriaHeliane) [inline CHANGELOG.md]: I think that it is better to put it in the `Fixed` section.

File: react/index.js [hunk 3]
```diff
@@ -238,36 +237,34 @@ class ProductSummary extends Component {
...
     } = this.props
 
-    const buyButtonClasses = classNames(
-      `${productSummary.buyButtonContainer} pv3 w-100`,
-      {
-        'dn': displayMode === 'small' || displayMode === 'inline',
-        'dn db-ns': displayMode === 'normal',
-      }
-    )
+    const buyButtonClasses = classNames(`${productSummary.buyButtonContainer} pv3 w-100 db`)
```
Comment (vitoriaHeliane) [inline react/index.js]: You don't need use `classNames` here, you can put it directly in the element's classname.

File: react/index.js [hunk 4]
```diff
@@ -238,36 +237,34 @@ class ProductSummary extends Component {
...
-    const showBuyButton = !equals(displayBuyButton, displayButtonTypes.DISPLAY_ON_HOVER.value) || mobile || this.state.isHovering
     const quantity = path(['sku', 'seller', 'commertialOffer', 'AvailableQuantity'], product) || 0
     const isAvailable = (quantity > 0)
 
     return (
-      !equals(displayBuyButton, displayButtonTypes.DISPLAY_NONE.value) && (
+      showBuyButton && (
         <div className={buyButtonClasses}>
-          <div className={`${productSummary.buyButton} center mw-100 ${!showBuyButton && 'isHidden'}`}>
+          <div className={`${productSummary.buyButton} center mw-100 ${!hoverBuyButton && productSummary.isHidden}`}>
```
Comment (vitoriaHeliane) [inline react/index.js]: It is better to extract it to a const and use `classNames`.

File: react/__mocks__/@vtex/buy-button.js [hunk 5]
```diff
@@ -1,3 +1,5 @@
+import React from 'react'
+
 export default function() {
-  return null
+  return <div></div>
```
Comment (luciannojunior) [inline react/__mocks__/@vtex/buy-button.js]: Just a hint, `<Fragment/>` would work

File: react/package.json [hunk 6]
```diff
@@ -30,15 +36,19 @@
   },
   "jest": {
     "verbose": true,
-    "testURL": "http://localhost/",
     "transform": {
+      "\\.(gql|graphql)$": "jest-transform-graphql",
```
Comment (luciannojunior) [inline react/package.json]: Do you know why is this necessary, since this app apparently doesn't use `.gql`?

Comment (theomoura) [reply react/package.json]: Dunno. I will remove and test to see if there is any colateral effect.
